librz/arch/meson.build

```diff
@@ -444,6 +444,8 @@ rz_arch_inc = [
 
 rz_arch_sources = arch_isa_sources + arch_il_sources + arch_esil_sources + arch_plugin_sources + arch_common_sources
 
+m_dep = cc.find_library('m', required: true)
```

Comment:
i think there is a check already in the main `meson.build` for libm. could you check how it is handled and add that dep directly?

Reply:
Indeed. Thanks.